host/windows/syscall.c

```diff
@@ -1488,13 +1502,14 @@ int oe_syscall_uname_ocall(struct oe_utsname* buf)
     // OE SDK is supported only on WindowsServer and Win10
     if (IsWindowsServer())
     {
-        sprintf(buf->sysname, "WindowsServer");
-        sprintf(buf->version, "2016OrAbove");
+        sprintf_s(buf->sysname, sizeof("WindowsServer"), "WindowsServer");
```

Comment:
the second argument to sprintf_s should be the size of the buffer (buf->sysname), not the size of the 3rd argument ("WindowsServer").  Compare line 1485 and 1493, which are correct.